### ISSUE TITLE
Character count component  aria describeby attribute bug

### ISSUE THREAD

**Issue by stephenjmcneill1971** [alphagov/govuk-design-system-backlog#287]
When a the character count is rendered on a text areas the aria-describeBy attribute lists the ID (suffixed with "-info") of the character count text and the ID of the hint text of the textarea (suffixed with "-hint") in that order. 

example:
**aria-describedby**="char-count-info q-textarea-hint"

This means that screen readers read the information referenced by those ID's in that order. 
Character count info read 1st
Textarea hint read 2nd

![image](https://github.com/alphagov/govuk-design-system-backlog/assets/119854428/687a8725-0ed0-4566-83f6-0554a3afffcb)

## What
> Need to have configuration so that the "-info" ID is appended after the textarea "-hint" ID. 

## Why
When using a screen reader the character count below the textarea is read out first before the hint/description text above the textarea. This is counter intuitive to the page it should be read in order of appearance to help those needing to use screen readers

**Comment by querkmachine** [comment]
As an initial, untested thought: Is this so that things are read out in that order as the user is typing into the textarea? 

In that context, hearing the updated character count would be of a higher priority than a repetition of the hint text, and the user is likely to want to hear that take place more frequently.

I'm unsure if that's actually the case, as I believe typing announcements are actually spawned from a visually-hidden live region, but something to confirm.
